Configuration/DataProcessing/test/RunPromptReco.py

```diff
@@ -196,6 +195,8 @@ def __call__(self):
             recoinator.writeDQMIO = True
         if opt == "--no-output":
             recoinator.noOutput = True
+        if opt == "--nThreads":
+            recoinator.nThreads = arg
```

Comment:
@andresfelquintero passing the option is not enough, it must be used within recoinator (which is not the case now)